kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue

```diff
@@ -93,6 +93,7 @@
           :value="true"
         />
       </div>
+
```

Comment:
Was the conflict just because we cherry-picked a fix from 0.12.y?

Reply:
I think it's because I did a bigger refactor on 0.12.y to add quiz preview to "Reports" pages, this is why the two versions are so radically different.